linera-client/src/chain_listener.rs

```diff
@@ -213,7 +213,7 @@ impl<C: ClientContext> ChainListener<C> {
             let admin_chain_id = guard.wallet().genesis_admin_chain();
             guard
                 .make_chain_client(admin_chain_id)
-                .synchronize_from_validators()
+                .synchronize_chain_state(admin_chain_id)
```

Comment:
Weird that we already have the client for `admin_chain_id` but the method needs this argument anyway. Can't it be loaded from the context of the chain client?

Reply:
Yes; both call `Client::synchronize_chain_state_from_committee` in the end.

Reply:
I meant: why not call `synchronize_chain_state_from_committee()` here instead?

Reply:
Sorry but this is dumb. We have:
- `synchronize_chain_state(chain_id)` which internally loads the committee and then synchronizes the state
- `synchronize_chain_state_from_committee(committee)` which requires committee but internally knows which chain to synchronize
- `synchronize_chain_state_from_committee` (on the client) that expects both

but we don't have `fn synchronize_chain_state` which hides both. Even though everything is available in the `ChainClient`: 
- which chain it's for
- how to get the committee

Reply:
I created https://github.com/linera-io/linera-protocol/issues/4715 for this. (It was not introduced in this PR.)